cfme/tests/test_rest.py

```diff
@@ -517,3 +519,39 @@ def test_edit_requests(self, rest_api, pending_requests, from_detail):
         for request in pending_requests:
             request.reload()
             assert request.options['arbitrary_key_allowed'] == 'test_rest'
+
+    @pytest.mark.uncollectif(lambda: current_version() < '5.7')
+    def test_create_automation_requests_parallel(self, rest_api):
+        """Create automation requests in parallel.
+
+        Metadata:
+            test_flag: rest
+        """
+        output = mp.Queue()
+        entry_point = rest_api._entry_point
+        auth = rest_api._auth
+
+        def _gen_automation_requests(entry_point, auth, output):
+            api = MiqApi(entry_point, auth, verify_ssl=False)
+            requests_data = automation_requests_data(
+                'nonexistent_vm', requests_collection=True, approve=False)
+            api.collections.requests.action.create(*requests_data[:2])
+            result = (api.response.status_code, api.response.json())
+            output.put(result)
+
+        processes = [
+            mp.Process(target=_gen_automation_requests, args=(entry_point, auth, output))
+            for _ in range(4)]
+
+        for proc in processes:
+            proc.start()
+
+        # wait for all processes to finish
+        for proc in processes:
+            proc.join()
```

Comment:
So I tried many different approaches only to find out that they would just complicate things or doesn't work at all. This is most straightforward working thing I was able to come up with.

@mshriver I tried to use Pool and map to make things more compact, but reverted it. In order to have anything meaningful to `map`, I would need to pass the outcome of `automation_requests_data`, like
```
requests = [automation_requests_data(
            'nonexistent_vm', requests_collection=True, approve=False)[:2] for _ in range(4)]
for status, response in pool.map(_gen_automation_requests, requests):
```
But I can't do that, because it isn't picklable. Also `map` would need `_gen_automation_requests` to be picklable, which is not (it would need to be top level function, not nested, which would also needlessly complicate things). Etc.